### ISSUE TITLE
docs: Document what "distro"/"version" are meant to be used for

### ISSUE THREAD

**Issue by iliana** [oxidecomputer/omicron#3108]
I think it's unclear to a user importing an image why they need to specify a distro and version. Does the rack use this metadata? Is it strictly for the sake of users?

**Comment by askfongjojo** [comment]
You are right that the rack (control plane and hypervisor) doesn't use this information at all. These attributes are included so that when the list image API supports filtering, they can be used as built-in search params, similar to cloud provider image search experience.

I'll add a note in the user guide to explain that these are documentation attributes.
